Button.cpp

```diff
@@ -0,0 +1,91 @@
+// Button.h
+// Sheng Fang, Rabiya Majeed, & Zulaikha Zakiullah
+// This instantiates the button class
+
+#include "Button.h"
+
+Button::Button(int w, int h) {
+
+	width = w;
+	height = h;
+
+	position.x = 0;
+	position.y = 0;
+
+	currentSprite = ButtonSprite::BUTTON_SPRITE_MOUSE_OUT;
+}
+
+Button::~Button() {}
+
+// Setters
+
+void Button::setPosition(int x, int y) {

```

Comment:
x and y coords are defined in Object.cpp. If you want it as an SDL_Point though I think you should do that in Object.cpp